knowledge/sports/athletes/jason_kelce/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Richard Hofmeister
+domain: athlete
+seed_examples:
+- answer: | Clevelan Heights, Ohio
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!